screens/login-screen.tsx

```diff
@@ -12,23 +13,63 @@ const LoginScreen = () => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
 
+  const validEmail = (email:string) => {
+    const er = /^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.com$/;
+    return er.test(email)
+  }
+
+  const validPassword = (password:string) => {
+    const er = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{7,50}$/;
+    return er.test(password)
+  }
+
+const loginHandle = (email:string, password:string) => {
+    if ( email.length == 0 || password.length == 0 ) {
+      Alert.alert('Campos obrigatórios!', 'Insira um e-mail e uma senha.', [
+        {text: 'Ok'}
+      ]);
+        return;
+    }
+    if ( password.length > 0 && password.length < 7 ) {
+      Alert.alert('Senha inválida!', 'A senha deve ter ao menos 7 caracteres.', [
+        {text: 'Ok'}
+      ]);
+      return;
+    }
+    if ( validEmail(email) == false) {
```

Comment:
Outra forma de verificar se é falso:

```suggestion
    if (!validEmail(email)) {
```

E é bom evitarmos usar `==` tbm, e usarmos `===` quando quisermos comparar igualdade. Qualquer dúvida dá uma pesquisada nos docs do javascript.